### ISSUE TITLE
How can FEI_ec_c be "coming soon" to TaxBrain?

### ISSUE THREAD

**Issue by martinholmer** [OpenSourcePolicyCenter/webapp-public#285]
@talumbau added `_FEI_ec_c` to the `TAXCALC_COMING_SOON_FIELDS` list in commit 7d873eb on January 27, 2016.  Unless I'm missing something, it seems to me that this was a mistake and that the `_FEI_ec_c` policy parameter should have been added to the `TAXCALC_HIDDEN_FIELDS` list.  

In the current version of Tax-Calculator, `_feided` is always zero (because there is no information on foreign earned income in the 2009 IRS-SOI PUF).  And, in addition, there is no information about whether Form 2555 was filed, and thus, the variable `f2555` is always zero in Tax-Calculator.  This situation implies that changes in the `_FEI_ec_c` policy parameter will have no effect on Tax-Calculator output, which is not what a TaxBrain user would expect if (s)he was offered the chance to change the `_FEI_ec_c` policy parameter.  Below is the only line in the functions.py file that uses this policy parameter:
```
c02700 = min(_feided, FEI_ec_c * f2555)
```
The value of `c02700` is always zero in the current version of Tax-Calculator, no matter what `_FEI_ec_c` value has been specified.

Also, note that all the other names in the `TAXCALC_COMING_SOON_FIELDS` list are not Tax-Calculator parameters, and therefore, should be deleted.

@MattHJensen @feenberg

**Comment by martinholmer** [comment]
@talumbau, the webapp/apps/taxbrain/helpers.py file contains the following code fragment:
```
TAXCALC_COMING_SOON_FIELDS = [
    '_Dividend_rt1', '_Dividend_thd1',
    '_Dividend_rt2', '_Dividend_thd2',
    '_Dividend_rt3', '_Dividend_thd3',
    '_BE_CG_trn', '_FEI_ec_c'
    ]

TAXCALC_HIDDEN_FIELDS = [
    '_ACTC_Income_thd',
    '_AMT_Child_em', '_AMT_em_pe', '_AMT_thd_MarriedS',
    '_CDCC_ps', '_CDCC_crt',
    '_DCC_c',
    '_EITC_InvestIncome_c', '_EITC_ps_MarriedJ',
    '_ETC_pe_Single', '_ETC_pe_Married',
    '_KT_c_Age',
    '_LLC_Expense_c'
]

TAXCALC_COMING_SOON_INDEXED_BY_MARS = [
    '_Dividend_thd1','_Dividend_thd2', '_Dividend_thd3'
]
```
But when I search for the phrase `Dividend_` in the Tax-Calculator repo, I find nothing:
```
tax-calculator$ grep -r --include="*py" Dividend_ .
tax-calculator$
tax-calculator$ grep -r --include="*json" Dividend_ .
tax-calculator$
```
And when I search for the phrase `BE_CG_trn` in the Tax-Calculator repo, I find nothing:
```
tax-calculator$ grep -r --include="*py" BE_CG_trn .
tax-calculator$ 
tax-calculator$ grep -r --include="*json" BE_CG_trn .
tax-calculator$ 
```
Am I missing something?  Or should those seven variables be removed from the `TAXCALC_COMING_SOON_FIELDS` list?  If so, then the `TAXCALC_COMING_SOON_INDEXED_BY_MARS` list should be empty.

And then what about the remaining `_FEI_ec_c` variable in the `TAXCALC_COMING_SOON_FIELDS` list?  Should it not be moved to the `TAXCALC_HIDDEN_FIELDS` list for the reasons mentioned earlier in this issue?

Perhaps I'm confused, but it seems as if the code here has fallen out of date.  The webapp team is putting a lot of energy into new interfaces for new OSPC models, so it would seem as if removing the highly misleading COMING SOON label on the `_FEI_ec_c` parameter should get at least a little consideration.

@MattHJensen @Amy-Xu

**Comment by talumbau** [comment]
In #137 , I interpreted Amy's comment:

"This parameter won't work right due to lack of data. So we should remove this one from input page for now."

to mean that we would, in a relatively short amount of time, have the ability to use this parameter. It looks like that is not the case. I will put `_FEI_ec_c` in the hidden fields.

**Comment by talumbau** [comment]
In commit https://github.com/open-source-economics/Tax-Calculator/commit/d455762009ef26351bba9da79143b96f71c796f6, @MattHJensen removed `BE_cg_trn`, so I'll remove that from the `TAXCALC_COMING_SOON_FIELDS`

**Comment by talumbau** [comment]
In commit https://github.com/open-source-economics/Tax-Calculator/commit/b8ca2850aa7cde955aa06aaeebc936d21f713c3c @Amy-Xu renamed the `Dividend_*` parameters, so I will delete any mention of these from the webapp repo.

**Comment by talumbau** [comment]
Re-opening so that this stays open until merge of #294. In Github, since I referred to this issue in #294 by writing "resolves #285", Github should automatically close this issue when PR #294 is merged.
